web/src/app/app.module.ts

```diff
@@ -2,7 +2,7 @@
  * Modules
  */
 import { BrowserModule } from '@angular/platform-browser';
-import { APP_INITIALIZER, NgModule } from '@angular/core';
+import { APP_INITIALIZER, ModuleWithProviders, NgModule } from '@angular/core';
```

Comment:
For what is `ModuleWithProviders` used for?

Reply:
This has been autoimported by Intelij and was a mistake
Has been removed in the upcoming commit